org.gnome.OTPClient.json

```diff
@@ -0,0 +1,83 @@
+{
+  "app-id": "org.gnome.OTPClient",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
```

Comment:
Any particular reason to target 3.24? 3.26 is the current version

Reply:
No :) last time I tried with 3.26 I got an error. Will try again (it's my first time using flatpak on the dev side)

Reply:
:+1:

Reply:
Since the upgrade to the 3.26 stack I can't run the app anymore:
```
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

any ideas?